style.css

```diff
@@ -0,0 +1,455 @@
+@charset "utf-8";
+
+html, body {
+    height: 100vh;     
+}
+
+a:link {text-decoration: none; color: white;}
+a:visited {text-decoration: none; color: white;}
+a:active {text-decoration: none; color: white;}
+a:hover {text-decoration: none; color: white;}
+    
+h2, h3 {
+    font-family: 'Playfair Display', serif;
+} 
+ .header-left {
+    float: left;
+    position: absolute;
+    top: 100px;
+    left: 50px;
+    color: #7CAF3F;
+    margin-left: 100px;
+}
+
+.header-right {
+    float: right;
+    position: absolute;
+    right: 100px;
+    top: 100px;
+    color: white;
+    display: block;
+    vertical-align: middle;
+    
+}
+
+.cart-img {
+    width: 50px;
```

Comment:
Be careful about using px in the next assignment. px are not responsive units and should not be used ANYWHERE in your css except to perhaps set the line width of an element.